### ISSUE TITLE
What is the most recent working version?

### ISSUE THREAD

**Issue by tenstriip** [melnijir/Dicaffeine#59]
Tried 32 bit and 64 bit versions of the most recent Raspbian OS as of 11/10/23 and nothing has worked.  Not sure what to do about these items:

Key issue:
"W: http://raspbian.raspberrypi.com/raspbian/dists/bookworm/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
W: https://dicaffeine.com/repository/dists/bullseye/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details."

"Setting up dicaffeine (0.6.3) ...
chown: cannot access '/etc/dhcpcd.conf': No such file or directory
dpkg: error processing package dicaffeine (--configure):
 installed dicaffeine package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 dicaffeine
E: Sub-process /usr/bin/dpkg returned an error code (1)"

**Comment by VindicoShaun** [comment]
I've managed to get the installer to work by using: `sudo apt install dhcpcd5` not sure what other repercussions this has

**Comment by tenstriip** [comment]
I was able to get further along by using the latest release of 32 bit Bullseye rather than Bookshelf, however the stream itself was not viewable. I tried sending from my desktop and another system, and while dicaffeine saw the inputs and correctly displayed source names, when I selected one and pressed play it would just black screen. I did not try streaming from the device, only to it as that is what I mainly need.
